CMakeLists.txt

```diff
@@ -61,6 +61,19 @@ IF(ENABLE_PAHO_MQTT)
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_MQTT=1")
 ENDIF()
 
+# AITT Library
+SET(ENABLE_AITT OFF)
```

Comment:
Get user option for AITT.
If user has explicitly enabled AITT and AITTLIB is not found --> error

e.g., https://tttsss77.tistory.com/193

Reply:
Thanks. I added options for MQTT and AITT. PTAL.